tcms/xmlrpc/api/testcase.py

```diff
@@ -990,21 +972,19 @@ def remove_tag(request, case_ids, tags):
     # Remove tag 'foo' and 'bar' from cases list '56789, 12345' with String
     >>> TestCase.remove_tag('56789, 12345', 'foo, bar')
     """
-    tcs = TestCase.objects.filter(
+    test_cases = TestCase.objects.filter(
         case_id__in=pre_process_ids(value=case_ids)
     )
-    tgs = TestTag.objects.filter(
+    test_tags = TestTag.objects.filter(
         name__in=TestTag.string_to_list(tags)
     )
 
-    for tc in tcs.iterator():
-        for tg in tgs.iterator():
+    for test_case in test_cases.iterator():
+        for test_tag in test_tags.iterator():
             try:
-                tc.remove_tag(tg)
+                test_case.remove_tag(test_tag)
             except ObjectDoesNotExist:
```

Comment:
Please verify if this is the `TestCase.remove_tag` method! If it is then it does a `QuerySet.filter().delete()` which doesn't raise ObjectDoesNotExist error I believe. In case of error let the exception propagate to the client so we can figure out what's going wrong.

Reply:
It is the same method. Done